staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -144,7 +142,7 @@ var (
 				4, 5, 6, 8, 10, 15, 20, 30, 45, 60},
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
-		[]string{"field_validation", "enabled"},
+		[]string{"field_validation"},
```

Comment:
update the description, since this no longer includes enabled status

Reply:
would be curious to hear from @logicalhan what the preferred approach for something like this is in the future, where we expect `enabled` to eventually be unconditionally true ... should that have been a separate metric, or omitted, or set to `true` unconditionally now?

Reply:
updated